.github/release-drafter.yml

```diff
@@ -0,0 +1,62 @@
+# Config for https://github.com/toolmantim/release-drafter
+name-template: 'Play $NEXT_MINOR_VERSION'
+tag-template: '$NEXT_MINOR_VERSION'
+filter-by-commitish: true
+commitish: master
+
+change-template: '- $TITLE [#$NUMBER](https://github.com/playframework/playframework/issues/$NUMBER) by [@$AUTHOR](https://github.com/$AUTHOR)'
+template: |
+  # :mega: Play $NEXT_MINOR_VERSION Released
```

Comment:
is this right?

compare with https://github.com/akka/akka/blob/master/.github/release-drafter.yml